src/main/kotlin/com/group/libraryapp/calculator/Calculator.kt

```diff
@@ -28,6 +28,10 @@ class Calculator(
 
   }
 
+  fun save(){
```

Comment:
⚠️ **[detekt]** <detekt.ContextOrder> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The function name to be checked